src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/NocRespondentHelper.java

```diff
@@ -31,100 +32,103 @@ public Map<String, Organisation> getRespondentOrganisations(CaseData caseData) {
-            .filter(respondent -> respondent.getValue().getRespondentName().equals(respName))
-            .findFirst().map(RespondentSumTypeItem::getValue)
-            .orElse(new RespondentSumType());
+                .filter(respondent -> respondent.getValue().getRespondentName().equals(respName))
+                .findFirst().map(RespondentSumTypeItem::getValue)
+                .orElse(new RespondentSumType());
     }
 
     public Optional<RespondentSumTypeItem> getRespondentSumTypeItem(CaseData caseData,
                                                                     RepresentedTypeRItem respondentRep) {
         final List<RespondentSumTypeItem> respondentCollection = caseData.getRespondentCollection();
         return respondentCollection.stream()
-            .filter(resp ->
-                resp.getValue().getRespondentName()
-                    .equals(respondentRep.getValue().getRespRepName())).findFirst();
+                .filter(resp ->
+                        resp.getValue().getRespondentName()
+                                .equals(respondentRep.getValue().getRespRepName())).findFirst();
     }
 
     public List<RepresentedTypeRItem> updateWithRespondentIds(CaseData caseData) {
-        return caseData.getRepCollection().stream()
-            .peek(respondentRep ->
-                getRespondentSumTypeItem(caseData, respondentRep)
+        List<RepresentedTypeRItem> repList = new ArrayList<>();
+        for (RepresentedTypeRItem respondentRep : caseData.getRepCollection()) {
+            getRespondentSumTypeItem(caseData, respondentRep)
                     .ifPresent(respondent ->
-                        respondentRep.getValue().setRespondentId(respondent.getId())))
-            .collect(toList());
+                            respondentRep.getValue().setRespondentId(respondent.getId()));
+            repList.add(respondentRep);
+        }
+        return repList;
     }
 
     public void amendRespondentNameRepresentativeNames(CaseData caseData) {
-        List<RepresentedTypeRItem> repCollection = emptyIfNull(caseData.getRepCollection()).stream()
-            .peek(respondentRep -> {
-                final List<RespondentSumTypeItem> respondentCollection = caseData.getRespondentCollection();
-                Optional<RespondentSumTypeItem> matchedRespondent = respondentCollection.stream()
+        List<RepresentedTypeRItem> repCollection = new ArrayList<>();
+        for (RepresentedTypeRItem respondentRep : emptyIfNull(caseData.getRepCollection())) {
+            final List<RespondentSumTypeItem> respondentCollection = caseData.getRespondentCollection();
+            Optional<RespondentSumTypeItem> matchedRespondent = respondentCollection.stream()
                     .filter(resp ->
-                        resp.getId().equals(respondentRep.getValue().getRespondentId())).findFirst();
+                            resp.getId().equals(respondentRep.getValue().getRespondentId())).findFirst();
```

Comment:
Nitpick but could `.findFirst()` jump down to the next line? (it wont let me add a suggestion here). Bonus points if this enables the filter statement to be on one line :D

```
collection.stream()
      .filter()
      .findFirst()
```